site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -218,6 +218,8 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `agones.allocator.generateTLS`                      | Set to true to generate TLS certificates or false to provide certificates in `certs/allocator/*`| `true`                 |
 | `agones.allocator.tolerations`                      | Allocator [toleration][toleration] labels for pod assignment                                    | `[]`                   |
 | `agones.allocator.affinity`                         | Allocator [affinity][affinity] settings for pod assignment                                      | `{}`                   |
+| `agones.allocator.disableTLS`                       | Turns off all authentication for incoming connections to the allocator.                         | `false`                |
```

Comment:
for `disableTLS`  how about turn off the built-in encryption and authentication mechanism for agones-allocator service.

Reply:
Changed!